cpp/arcticdb/storage/azure/azure_storage.cpp

```diff
@@ -291,7 +292,12 @@ AzureStorage::AzureStorage(const LibraryPath &library_path, OpenMode mode, const
     Storage(library_path, mode),
     root_folder_(object_store_utils::get_root_folder(library_path)),
     request_timeout_(conf.request_timeout() == 0 ? 60000 : conf.request_timeout()) {
-        azure_client_ = std::make_unique<RealAzureClient>(conf);
+        if(conf.use_mock_storage_for_testing()) {
+            azure_client_ = std::make_unique<MockAzureClient>();
+        }
+        else {
```

Comment:
else on the line above to match our normal formatting

Reply:
Done in [ab07c4a](https://github.com/man-group/ArcticDB/pull/1331/commits/ab07c4a088ffabddde3a0c43103586332957ab1f)